lib/connect/index.js

```diff
@@ -87,6 +90,7 @@ function connect (brokerUrl, opts) {
           case 'wx':
             opts.protocol = 'wxs'
             break
+          case 'alis':
```

Comment:
I fear this might throw for alis.

Reply:
case 'ali':
            opts.protocol = 'alis'
            break